scripts/clean_validate.py

```diff
@@ -0,0 +1,95 @@
+# clean_validate.py
+# author: Weilin Han
+# date: 2024-10-20
+
+import click
+import os
+import sys
+import re
+import pandas as pd
+import great_expectations as gx
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from src.clean_data import *
+from src.validate_data import *
```

Comment:
From PEP8: "Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools."

So we should explicitly import all 12 functions.